pubspec.yaml

```
@@ -101,3 +102,5 @@ flutter:
         - asset: fonts/custom-icons.ttf
   assets:
     - assets/i18n/
+flutter_localizations: # add this
+    sdk: flutter
```

Two spaces, not four. grrr

```suggestion
  sdk: flutter
```